syntax = "proto3";

package CodecServer.proto;

message Handshake {
  string serverName = 1;
  string serverVersion = 2;
}